test_client.py

```diff
@@ -20,15 +35,17 @@
     elif event.type == enet.EVENT_TYPE_RECEIVE:
         print("%s: IN:  %r" % (event.peer.address, event.packet.data))
         continue
+
     msg = os.urandom(40)
     packet = enet.Packet(msg)
     peer.send(0, packet)
 
     counter += 1
     if counter >= MSG_NUMBER:
-        msg = SHUTDOWN_MSG
```

Comment:
Ok. Wanted to do it after the loop, that's why it's going during last iteration. Will make it in another loop.